### DISCUSSION TITLE
79003841-E8FA-6989-927D-98D06F85666B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#134] [General]
# 79003841-E8FA-6989-927D-98D06F85666B

50000 Quaoar is a dwarf planet in the Kuiper belt that measures approximately 1,110 km in diameter, or about half the size of Pluto. It hosts a single moon named Weywot and two narrow rings which both orbit outside of Quaoar's Roche limit. The outer ring is highly asymmetric and clumpy, with a radial width ranging 5-300 km.

(NOTE: This modifies the stock version of Quaoar, but a restart of Celestia is still needed for changes to work)

https://celestia.mobi/addon?item=79003841-E8FA-6989-927D-98D06F85666B

**Comment by Lil-Gdops** [comment]
Broken

**Reply by SevenSpheres** [reply]
What exactly is broken?

**Reply by SphereOfLife** [reply]
It is like a black box where the rings should be.

**Comment by BESTBOY12345** [comment]
How do you make mods on here??
